ui-v2/app/styles/components/tabular-collection.scss

```diff
@@ -16,6 +16,17 @@ table tr > * {
 tr > * dl {
   float: left;
 }
+/* TODO: putting this here is less than ideal */
+/* but this is another area where I am specifically */
+/* targetting table-like things. This is now a prime */
+/* area for a bit of refactoring/reorganizing */
+html.template-service.template-list td:first-child a span,
+html.template-node.template-show #services td:first-child a span {
+  @extend %with-external-source-icon;
+  float: left;
+  margin-right: 10px;
+  margin-top: 2px;
+}
```

Comment:
This is going to be all split out elsewhere, next time I do a full CSS review/refactor. I'll probably make a '%service-row' component or similar, again best done in a future PR.